docs/schema/manifest.schema.json

```diff
@@ -0,0 +1,419 @@
+{
+    "definitions": {
```

Comment:
This is missing a meta-schema. It needs a `$schema` property set to `http://json-schema.org/draft-07/schema#` or `http://json-schema.org/draft-07/schema#` or some other version